src/data_gradients/managers/abstract_manager.py

```diff
@@ -257,3 +259,34 @@ def _create_samples_iterated_warning(self) -> str:
         msg_train = f"Train set: {self._train_iters_done} out of {total_train_samples} samples were analyzed{portion_train}.\n"
         msg_val = f"Validation set: {self._val_iters_done} out of {total_val_samples} samples were analyzed{portion_val}.\n "
         return msg_head + msg_train + msg_val
+
+    @staticmethod
+    def _get_grouped_feature_extractors(default_config_name: str, config_path: str, feature_extractors) -> Dict[str, List[AbstractFeatureExtractor]]:
+        if feature_extractors is None:
+            if config_path is None:
+                config_dir, config_name = None, default_config_name
+            else:
+                config_path = os.path.abspath(config_path)
+                config_dir, config_name = os.path.dirname(config_path), os.path.basename(config_path).split(".")[0]
+            grouped_feature_extractors = load_report_feature_extractors(config_name=config_name, config_dir=config_dir)
+        else:
+            if not isinstance(feature_extractors, list):
+                feature_extractors = [feature_extractors]
+
+            section_name = "Selected features"
+            grouped_feature_extractors = {section_name: []}
+            for feature_extractor in feature_extractors:
+                if isinstance(feature_extractor, AbstractFeatureExtractor):
+                    grouped_feature_extractors[section_name].append(feature_extractor)
+                elif isinstance(feature_extractor, str):
+                    grouped_feature_extractors[section_name].append(FeatureExtractorsFactory().get(feature_extractor))
+                elif issubclass(feature_extractor, AbstractFeatureExtractor):
+                    try:
+                        grouped_feature_extractors[section_name].append(feature_extractor())
+                    except RuntimeError as e:
+                        raise RuntimeError(f"The feature extractor {feature_extractor.__class__} requires additional init argument. "
+                                           f"Initialize the feature extractor and pass it as an instance")
```

Comment:
I recently started to add `from e` at the end of custom exceptions, because it will also provide the original exception, but your message will still be the main one and will appear at the bottom of the stack. Eventually, it just helps having a more informative error stack.

e.g. `raise TypeError(f"{e}\n\t => Could not load `{cls.__name__}` from cache.") from e`

I think it would be nice to have it here to let the user know what exactly broke, and what params were missing.